portlet/vue-portlet-webpack/.eslintrc.json

```diff
@@ -0,0 +1,87 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "jest": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:vue/recommended",
+        "plugin:jest/recommended"
+    ],
+    "globals": {
+        "Vue": true,
+        "eXo": true,
+        "$": true,
+        "global": true,
+        "exoi18n": true,
+        "ExtensionRegistry": true
```

Comment:
The ExtensionRegistry is not available yet, it should be added in the sample once it is integrated in the product IMO.

Reply:
Done